index.html

```diff
@@ -1,4 +1,5 @@
-<html>
+<!DOCTYPE html>
+
```

Comment:
Good catch adding the DOCTYPE, as it is best practice / required. Looks like the opening <html> tag may have bene accidentally deleted.